### ISSUE TITLE
Documentation of the helpers and where/when to use them

### ISSUE THREAD

**Issue by celluj34** [marcominerva/TinyHelpers#74]
Hello;

I think it would be helpful in the README to document each of the helpers, what they do, and where to use them. It looks like there's a lot of neat things in here but discoverability isn't very good.

**Comment by marcominerva** [comment]
You are absolutely right, unfortunately writing documentation is a long task. I have planned to write it, I need to find the time :-)

**Comment by marcominerva** [comment]
Yes, it's a good idea! Thank you for your suggestion!

**Comment by vikramvee** [comment]
Hi @marcominerva, Please let me know if I can be of any help here. This is a great library and could be helpful for many.

**Comment by marcominerva** [comment]
Thank you @vikramvee! 

The are two main areas for which a contribution will be great:

- Add XML documentation for **TinyHelpers.AspNetCore**, **TinyHelpers.EntityFrameworkCore** and **TinyHelpers.Dapper** libraries (currently, only the methods of the main **TinyHelpers** library is documented using XML)
- Make the documentation directly available within GitHub. At this moment, I have done this: https://github.com/marcominerva/TinyHelpers/tree/master/docs/TinyHelpers, that is created using the .NET tool available at https://github.com/ejball/XmlDocMarkdown, but the are some problems with .NET 7.0 and higher assemblies, so I need to check other solutions (for example creating an ad-hoc application like described here: https://ejball.com/XmlDocMarkdown/).
